Jenkinsfile

```diff
@@ -61,7 +58,7 @@ pipeline {
                                             sh 'make build'
                                             sh 'make test'
                                             // If the tests are passing for Linux AMD64, then we can build all the CPU architectures
-                                            sh 'docker buildx bake --file docker-bake.hcl linux'
+                                            sh 'make buildall'
```

Comment:
```suggestion
                                            sh 'make build-all'
```

nit: use a dashed case as word separator to be homogeneous with the ci-init or check-reqs targets

Reply:
We can't use `build-all` unfortunately as there is another make target `build-%` just above already catching every call to `make build-`*.

Reply:
WDYT of `complete-build`?

Reply:
Or "everything"? `make everything`

Reply:
Or `every-build`?

Reply:
After discussing it together, `every-build` seems the best, avoiding the issue of the "build-" prefix already taken and keeping the intent clear.